ads/fluct.md

```diff
@@ -0,0 +1,35 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# fluct
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="fluct"
+    data-g="{GROUP-ID}"
+    data-u="{UNIT-ID}">
+</amp-ad>
+```
+
+## Configuration
+
+For more information, please [contact us](https://corp.fluct.jp/en/contact.php).
+
+Supported parameters:
```

Comment:
From your code, `data-g` and `data-` should be mandatory. please indicate they're required in the doc.